eland/operations.py

```diff
@@ -1439,8 +1439,7 @@ def update_query(self, boolean_filter):
         task = BooleanFilterTask(boolean_filter)
         self._tasks.append(task)
 
-    @staticmethod
-    def quantile_to_percentile(quantile: Any) -> float:
+    def quantile_to_percentile(self, quantile: Union[int, float]) -> float:
```

Comment:
Sorry for being confusing, didn't mean to change this into a method (ie a function that has `self`). I meant change it into a global function within the `eland/operations.py` file.

Reply:
Yep got it. 💯